### ISSUE TITLE
Metadata display options

### ISSUE THREAD

**Issue by noah-schechter** [atlosdotorg/atlos#404]
TLDR: I really like the color and icons on Metadata but I think we should consider not adding color to the pane until the incident page overhaul.

It looks great right now, but is redundant to the information displayed at the top of the page. Without the color, the Metadata pane feels like an input, where information displayed at the top is an output. With the color, this workflow feels less straightforward.

Similarly, we now use two colors (the more intense and less intense shades) to communicate the same information on incident pages. This is a tough problem because the bright colors look great at the top and the pastels are better on the body of the page. I think we bypass this problem by not shipping a colorful Metadata pane until the incident page overhaul.
